### PR TITLE
SF-677: add saber token list registry

### PR DESCRIPTION
**Pull request by aaronovz1**

### CODE REVIEW

File: src/utils/tokens.ts [hunk 1]
```diff
@@ -66,6 +66,28 @@ export const getTokenMap = async (
     }
   }
 
+  const rawSaberTokenList = (
+    await axios.get<any>(
+      "https://registry.saber.so/data/token-list.mainnet.json"
+    )
+  ).data.tokens as TokenInfo[];
```
Comment (ebramanti) [inline src/utils/tokens.ts:73]: Instead of casting we should type it with axios, think it's `{ tokens: TokenInfo[] }`

File: src/utils/tokens.ts [hunk 2]
```diff
@@ -79,6 +101,8 @@ export const getTokenMap = async (
     }
   }
 
+  console.log("tokenMap", tokenMap);
```
Comment (ebramanti) [inline src/utils/tokens.ts:104]: Please remove

File: src/utils/tokens.ts [hunk 3]
```diff
@@ -66,6 +66,28 @@ export const getTokenMap = async (
...
+  const rawSaberTokenList = (
+    await axios.get<any>(
+      "https://registry.saber.so/data/token-list.mainnet.json"
+    )
+  ).data.tokens as TokenInfo[];
+  const saberTokenList = new TokenListContainer(rawSaberTokenList);
+  const saberTokenInfos = saberTokenList
+    .filterByClusterSlug(cluster)
+    .excludeByTag("saber-stableswap-lp")
+    .excludeByTag("saber-hidden")
```
Comment (ebramanti) [inline src/utils/tokens.ts:78]: Nice one

File: src/utils/tokens.ts (hunk 2 above)
Comment (aaronovz1) [reply src/utils/tokens.ts:104]: whoooops

**Review by ebramanti — APPROVED** [review]

**Pull request merged**